### ISSUE TITLE
Error on Xcode 9.2

### ISSUE THREAD

**Issue by MrShft** [JSSAlertView/JSSAlertView#74]
Hi there,

I installed this pod on a project in Xcode 9.2 and kept getting errors at the import line. It gave 2 different kinds of errors:

> _No such module 'JSSAlertView'_

> _Cannot load underlying module 'JSSAlertView'_

Some background details:
- The project was fresh, with some UI elements set up in Storyboard and no other code in ViewController.
- The 'Firebase/Core' pod was successfully installed right before it.

Is there an issue with JSSAlertView with Xcode 9.2/Swift 4? CocoaPods says it's compatible.

Please advise.

Thanks in advance.

**Comment by MrShft** [comment]
Of course, I found the solution right after I made my post...

Here's what worked for anyone else who may experience my problem:

_1. I removed all pods (via CocoaPods) from my project, closed the Project and quit Xcode._
_2. Then, I opened the original xcodeproj file and ran the app, building it on to my test device successfully._
_3. Then, I quit Xcode._
_4. Through CocoaPods, I then installed the Firebase pods and JSSAlertView._
_5. I opened the Xcode xcworkspace file that CocoaPods created._
_6. I ran the app again, building it on to my test device..._

**_SUCCESS!_**

Hopefully, that helps anyone else out who may experience the same issues that I did.

Back to my code!
